robottelo/ui/locators/menu.py

```diff
@@ -4,6 +4,16 @@
 from selenium.webdriver.common.by import By
 from .model import LocatorDict
 
+# This is the old menu container xpath, was failing in upstream
+# kept here for reference
+# MENU_CONTAINER_XPATH = (
+#     "//div["contains(@style,'static') or contains(@style,'fixed')]"
+# )
+
+# Downstream has 2 elements with id=menu we should get last()
+# which is the active menu
+MENU_CONTAINER_XPATH = '(//ul[@id="menu"])[last()]'
```

Comment:
@oshtaier as long as we have elements with `id="menu"` and `id="menu2"` I don't get why it is risky?  it will always find at least one. 

I see a problem regarding the fact that there are elements named with same id twice in the UI, this is an error or a workaround.